research.html

```diff
@@ -159,6 +159,21 @@ <h3 class="h3">Flexible Energy Demand</h3>
             </div>
           </div>
         </div>
+        <div class="h-flex mb-150 left">
+          <div class="pr-150">
+            <a href="https://ie3.etit.tu-dortmund.de/research/third-party-projects/" class="link-block w-inline-block"><img src="images/ReCoDE-Logo.png" loading="lazy" alt="" class="project-logo"></a>
```

Comment:
Would be great if Recode would get an entry at our ie-3 website as well and then would be linked here. Is that planned to do?

Reply:
Now I added the correct link.